Source/ThirdParty/PlayCppSdkLibrary/Include/defi-wallet-core-cpp/src/contract.rs.cc

```diff
@@ -20,19 +20,23 @@ inline namespace cxxbridge1 {
 
 #ifndef CXXBRIDGE1_PANIC
 #define CXXBRIDGE1_PANIC
-template <typename Exception> void panic [[noreturn]] (const char *msg);
+template <typename Exception>
+void panic [[noreturn]] (const char *msg);
```

Comment:
is this some intended or unintended formatting change?

Reply:
Since I updated the bindings, I simply replaced the bindings, therefore it is the original format.